scrape.py

```diff
@@ -46,10 +47,19 @@ def bulletPointScrape(arr, link, date, category):
     curItem['blurb'] = blurbText
     curItem['links-within-blurb'] = links
     curItem['article-link'] = link
+    curItem['states'] = getStates(curItem).regions
+    curItem['cities'] = getStates(curItem).cities
```

Comment:
the array brackets `[` make this a little harder to work with in the spreadsheet. lets convert them into a string with a comma separated list. 

We can do something like this:
`curItem['states'] = ",".join(str(x) for x in curItem['states'])`

Lets do this for, `cities`, `regions` and `links-within-blurb`